ui/src/data-services/models/capture-details.ts

```diff
@@ -14,6 +15,10 @@ export class CaptureDetails extends Capture {
     }
   }
 
+  get canUpdate(): boolean {
```

Comment:
@mihow I felt a bit weird checking for update permission on capture, we should perhaps check for create permission on jobs? Although I rather avoid fetch a list of jobs, just to get this info.

Let me know if this solutions is fine or if you have thoughts :)